### PR TITLE
Fix Endpoints labels

### PR DESCRIPTION
**Pull request by CyberDem0n**

### CODE REVIEW

File: postgres-appliance/spilok8s.yaml [hunk 1]
```diff
@@ -85,7 +85,8 @@ kind: Endpoints
 metadata:
   name: &cluster_name spilodemo
   labels:
-    app: *cluster_name
+    application: spilo:
+    spilo-cluster: *cluster_name
```
Comment (a1exsh) [inline postgres-appliance/spilok8s.yaml]: So we now have camelCase, snake_case, and now lisp-case or what? ;-)

File: postgres-appliance/spilok8s.yaml [hunk 2]
```diff
@@ -85,7 +85,8 @@ kind: Endpoints
 metadata:
   name: &cluster_name spilodemo
   labels:
-    app: *cluster_name
+    application: spilo:
```
Comment (ikitiki) [inline postgres-appliance/spilok8s.yaml]: Should there be a colon in the end?

Comment (CyberDem0n) [inline postgres-appliance/spilok8s.yaml]: Definitely not, thank you, fixed

File: postgres-appliance/spilok8s.yaml [hunk 3]
```diff
@@ -85,7 +85,8 @@ kind: Endpoints
 metadata:
   name: &cluster_name spilodemo
   labels:
-    app: *cluster_name
+    application: spilo
+    spilo-cluster: *cluster_name
```
Comment (a1exsh) [inline postgres-appliance/spilok8s.yaml]: Why dash in `spilo-cluster`?  We already have camelCase and snake_case there...

Comment (CyberDem0n) [inline postgres-appliance/spilok8s.yaml]: Because it is already 'spilo-cluster' in all other places. Moreover, we have 'spilo-role' label.

Comment (a1exsh) [inline postgres-appliance/spilok8s.yaml]: That's not very consistent.

**Comment by alexeyklyukin** [comment]
👍

File: postgres-appliance/spilok8s.yaml (hunk 3 above)
Comment (CyberDem0n) [inline postgres-appliance/spilok8s.yaml]: Not consistent with what? camelCase is used for kubernetes objects and entities, we are using dashes for our objects (spilo-cluster, spilo-role).
And `cluster_name` is a very special case, it is used and the name for a yaml anchor and alias.

**Comment by CyberDem0n** [comment]
:+1:

**Pull request merged**